feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -30,7 +30,6 @@ import com.revenuecat.purchases.PurchasesErrorCode
 import com.revenuecat.purchases.common.BillingAbstract
 import com.revenuecat.purchases.common.ReplaceSkuInfo
 import com.revenuecat.purchases.common.caching.DeviceCache
-import com.revenuecat.purchases.common.firstSku
```

Comment:
I did a quick test on this branch and I think this extension is still needed. Otherwise `BillingWrapperTest` won't compile. It's using the extension from  `purchaseHistoryRecordExtensions.kt` it seems.